src/app/components/CollapsibleNavigation/constants.tsx

```diff
@@ -0,0 +1,253 @@
+import { CollapsibleNavigationSection } from './types';
+
+const CollapsibleNavigationSections: CollapsibleNavigationSection[] = [
+  {
+    id: 'home',
+    title: 'Home',
+    href: 'https://www.bbc.com',
```

Comment:
Should the home link return users to this page, like it does for all other homepages? Or is that different to the current behaviour?
```suggestion
    href: 'https://www.bbc.com/ws/languages',
```

Reply:
On the current page - https://www.bbc.com/ws/languages, Home goes to https://www.bbc.com/news?

Reply:
I asked Eva Theodoridou about this and we agreed to remove any links until Thursdays demo

<img width="612" height="111" alt="Screenshot 2025-08-19 at 14 49 46" src="https://github.com/user-attachments/assets/6f1b3e80-d8e6-41cf-89ee-3816f0537d43" />